### PR TITLE
comment-commands workflow: stop checkout

### PR DESCRIPTION
**Pull request by hexagonrecursion**
#### Summary
None

#### Purpose of change

actions/checkout downloads the source code from CleverRaven/Cataclysm-DDA. As far as I can tell this is completely unnecessary for this workflow

#### Describe the solution

remove redundant workflow step

#### Testing

None

### CODE REVIEW

**Comment by casswedson** [comment]
TL;DR this step is needed to fetch the repo's config, it's a very important step

this file btw https://github.com/CleverRaven/Cataclysm-DDA/blob/master/.github/comment-commands.yml

I guess making a shallow clone of the repo to get this one file is overkill; really, any other method is probably not "safe", wgeting it or something may just be overcomplicating it too much

I have a test repo for this, I replicated this commit in https://github.com/casswedson/issue-labeling/commit/6afe236f74ea06634c6167b100307f12f0081d13 made a comment in an issue suitable for testing https://github.com/casswedson/issue-labeling/issues/12#issuecomment-1153222577 the action fails silently cause it does not have any instructions to execute https://github.com/casswedson/issue-labeling/actions/runs/2484081971

**Comment by hexagonrecursion** [comment]
thanks for clarification
